Added ICD503 Templates

mshoger opened the pull request.

mshoger committed:
Added ICD503 Templates

shawndwells commented on opencontrol.yaml:
```diff
@@ -4,8 +4,12 @@ metadata:
   - ./dhs-4300a.yaml
+  - ./ICD503-low.yaml
+  - ./ICD503-moderate.yaml
+  - ./ICD503-high.yaml
```
- Because ICD 503 has a space in the name, should these files be ``icd-503-*``?
- Filenames should be lowercase

shawndwells commented on ICD503-high.yaml:
```diff
@@ -0,0 +1,404 @@
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD503-High
+standards:
```
two extra spaces are needed starting on line 6...

``6,$s/^/  /r``

shawndwells commented on ICD503-low.yaml:
```diff
@@ -0,0 +1,181 @@
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD503-Low
+standards:
```
two extra spaces, starting on line 6

shawndwells commented on ICD503-moderate.yaml:
```diff
@@ -0,0 +1,321 @@
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD503-Moderate
+standards:
```
two extra spaces starting here too :)

shawndwells reviewed (changes requested):
Thanks @mshoger! Very small adjustments needed.

- Lowercase file names (IDC503-high --> icd503-high)

- Because ICD 503 has a space in the name, should the file names too? e.g. ``icd-503-high.yaml``? If so, need to also adjust the ``name`` element in the files

mshoger committed:
Update per standards

Renamed files to lowercase and separated, updated opencontrol.yaml.

mshoger commented:
OK, the latest update has the files renamed per standard (good catch!) and the opencontrol.yaml updated to reflect that. Can you re-check the spacing? I just pulled up the 3 yaml files side by side with the fedramp and DHS ones and the spacing appears to be the same.

I may be completely missing that too...

shawndwells commented on icd-503-high.yaml:
```diff
@@ -0,0 +1,404 @@
+# Current as of 15-June-2018
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD503-High
```
``name`` is a human readable field. Would ``ICD 503 High`` be appropriate? Or ``ICD 503 High-High-High``? I don't know what the appropriate nomenclature would be.

shawndwells replied:
(same applies to the ``moderate`` and ``low`` profiles)

shawndwells reviewed (changes requested):
Minor change - the ``name`` fields should be updated too (across all the profiles).

You're right about the spaces. I was seeing things!

shawndwells commented:
PR resolves https://github.com/opencontrol/certifications/issues/2

mshoger committed:
Update icd-503-low.yaml

Changed name field to customer standard reference.

mshoger committed:
Update icd-503-high.yaml

Changed name field to customer standard reference.

mshoger committed:
Update icd-503-moderate.yaml

Changed name field to customer standard reference.

mshoger commented:
Hmmm. Good point. Changed name field to customer standard reference. Using L-L-L, etc. which is how they are referred to within customer documentation and standards.

shawndwells commented on icd-503-high.yaml:
```diff
@@ -0,0 +1,404 @@
+# Current as of 15-June-2018
+# Maintained by Mark Shoger (mshoger@redhat.com)
+# https://www.dni.gov/files/documents/ICD/ICD_503.pdf
+name: ICD-503 H-H-H
```
Should the dash in the names be kept? Consistently seeing it referred to as ``ICD 503`` vs ``ICD-503``

ref: 
https://fas.org/irp/dni/icd/icd-503.pdf

mshoger committed:
Update icd-503-high.yaml

Removed dash to be consistent with ODNI nomenclature.

mshoger committed:
Update icd-503-low.yaml

Removed dash to be consistent with ODNI nomenclature.

mshoger committed:
Update icd-503-high.yaml

Removed dash to be consistent with ODNI nomenclature. Remembered to edit line 1 this time too...

mshoger committed:
Update icd-503-moderate.yaml

Removed dash to be consistent with ODNI nomenclature.

mshoger commented:
Yep, you're absolutely right. Removed dash to be consistent with ODNI nomenclature.

shawndwells commented:
Thanks @mshoger! Merged.
